ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -5,7 +5,7 @@
 
 declaration template components/ssh/schema;
 
-include 'quattor/types/component';
+include 'quattor/schema';
```

Comment:
@ajf8 yes, it got refactored completly, because we want to unittest templates at some point, and minimal includes are a lot better.
in the same effort, the bind statement at the bottom should be moved to the config file (but you aren't touching that, so it's ok for now).